File: URI-Online-Judge/1069_dimonds_and_sand.py
test = int(input())

while test:
    string = input()
    dimonds = 0
    less_than = 0
    for c in range(len(string)):
        if string[c] == "<":
            less_than += 1

        if string[c] == ">" and less_than > 0:
            dimonds += 1
            less_than -= 1

    print(dimonds)
    test -= 1

File: URI-Online-Judge/1069_dimonds_and_sand.cpp
#include <iostream>
#include <string>


int main(){

	std::string line;
	int test;
	std::cin >> test;
	while(test--){
		std::cin >> line;
		int smaller_than = 0, dimonds = 0;
		int i = 0;
		while(line[i]){
			if(line[i] == '<')	
				smaller_than++;
			if(line[i] == '>' && smaller_than > 0){
				dimonds++;
				smaller_than--;
			}
			i++;
		}
		std::cout << dimonds << std::endl;
	}
	return 0;
}
	

	

